src/Extension/MemberExtension.php

```diff
@@ -2,22 +2,26 @@
 
 namespace SilverStripe\SessionManager\Extensions;
 
-use SilverStripe\Control\Session;
-use SilverStripe\Core\Extension;
 use SilverStripe\Forms\FieldList;
-use SilverStripe\Forms\GridField\GridField;
-use SilverStripe\Forms\GridField\GridFieldConfig_Base;
-use SilverStripe\ORM\FieldType\DBDatetime;
-use SilverStripe\Security\Permission;
-use SilverStripe\Security\PermissionProvider;
-use SilverStripe\Security\Security;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\Security\Member;
 use SilverStripe\SessionManager\FormField\SessionManagerField;
-use SilverStripe\SessionManager\Forms\GridFieldRevokeLoginSessionAction;
 use SilverStripe\SessionManager\Model\LoginSession;
 
-class MemberExtension extends Extension implements PermissionProvider
+/**
+ * Augment `Member` to allow relationship to the LoginSession DataObject
+ * @method LoginSession[]|DataList LoginSessions()
+ * @method Member getOwner()
+ */
+class MemberExtension extends DataExtension
 {
-    public const SESSION_MANAGER_ADMINISTER_SESSIONS = 'SESSION_MANAGER_ADMINISTER_SESSIONS';
+    /**
+     * URL to the user help abot managing session
+     * @var string
+     * @config
+     */
+    private static $session_login_help_url =
+        'https://userhelp.silverstripe.org/en/4/optional_features/managing_devices';
```

Comment:
This might need to be changed later https://github.com/silverstripe/silverstripe-session-manager/issues/15

Reply:
There's precedent for making it configurable. The user help menu we added to the left CMS column is configurable.

I'm not sure I'm a big fan of adding inline help links in the CMS ... if we do this everywhere, we'll end up with hundreds of links. If that's actually what we are aiming for, that's fine. Let's be systematic about it and agree on a standardised way of doing it.

If this is a special use case that we don't expect will repeat again, than maybe it doesn't matter all that much how we go about it.

@silverstripeux Got any views on this? ^

Reply:
I think the rational for the user help menu being configurable was so that people could write their own end user docs if they had a very custom CMS?  (I'm far from 100% on that though).  As I said previously, I cannot see a downstream project ever changing this config, so I'd prefer not to have the API addition of making this configurable.

Making it a private const means more flexibility to change it later or remove entirely if we decide we don't want to hardcode userhelp links in the codebase.

Reply:
I'll set it back to a non-configurable string, but I would still like to hear @silverstripeux on the wider issue. Is this a pattern we expect to see more frequently?

Reply:
Chiming in... In regards to precedents, the MFA module is a better example to refer to. Here's that implementation, let's follow that:  
[https://github.com/silverstripe/silverstripe-mfa/blob/f808ca51bdc4c064eb6d8c9c789eb8bb31b0bb61/src/Extension/SiteConfigExtension.php#L29](https://github.com/silverstripe/silverstripe-mfa/blob/4/src/Extension/SiteConfigExtension.php#L29)

Reply:
I'm personally okay with it being non-configureable when it is a one off link that is unlikely to change. I think if the experience changed considerably, it's easy enough to hide that link with CSS and for devs to provide guidance in other ways. This isn't the way I expect links to User Help to be be presented all over the CMS but we would need to develop a proper component to be able to utilise anywhere in the CMS.

> I think the rational for the user help menu being configurable was so that people could write their own end user docs if they had a very custom CMS?

Yes correct, people could provide links to additional docs, replace our links e.g. add the dev agency link for support.

Reply:
The MFA precedent is to make it configurable. So this would imply keeping the current behaviour as-is.